script.js

```diff
@@ -0,0 +1,21 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('#mobile-navbar');
+const mobileMenuList = document.querySelector('#mobile-menu-list');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  mobileMenuList.classList.toggle('hide-mobile-menu');
+});
+
+document.querySelectorAll('.hide-mobile-menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  mobileMenuList.classList.remove('hide-mobile-menu');
+}));
+
+document.querySelectorAll('.hide-mobile-menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.add('active');
+  navMenu.classList.add('active');
+  mobileMenuList.classList.add('hide-mobile-menu');
+}));
```

Comment:
- [ ] **[OPTIONAL]:** Sorry that this wasn't pointed out by the previous reviewer, but your menu currently has a bug. 🐛 When I click on a link, it does get me to the correct section, however, the hamburger icon stays as an "X" as shown below:

![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/77069011/202031175-c37be8b4-8ec8-4d85-9938-965f8a93febf.gif)

Also, you have two loops for the same collection of items, remember that keeping your code clean and not repeating yourself is an important concern for code maintainability! 🤖 

I suggest the following changes, please consider applying them! 🤓 🛠️ 

```suggestion
document.querySelectorAll('.hide-mobile-menu').forEach((n) => n.addEventListener('click', () => {
  hamburger.classList.remove('active');
  navMenu.classList.remove('active');
  mobileMenuList.classList.add('hide-mobile-menu');
}));
```

Reply:
I have resolved this also. Thank you @anagudelogu ``

`
`const hamburger = document.querySelector('.hamburger');
const navMenu = document.querySelector('#mobile-navbar');
const mobileMenuList = document.querySelector('#mobile-menu-list');

hamburger.addEventListener('click', () => {
  hamburger.classList.toggle('active');
  navMenu.classList.toggle('active');
  mobileMenuList.classList.toggle('hide-mobile-menu');
});

document.querySelectorAll('.hide-mobile-menu').forEach((n) => n.addEventListener('click', () => {
  hamburger.classList.remove('active');
  navMenu.classList.remove('active');
  mobileMenuList.classList.remove('hide-mobile-menu');
}));`
```